deployment/lightsail/main.tf

```diff
@@ -0,0 +1,51 @@
+provider "aws" {
+  region = var.region
+}
+variable "region" {
+  type = string
+}
+variable "whitefox_token" {
+  type = string
+}
+resource "aws_lightsail_container_service" "whitefox-container-service" {
+  name        = "whitefox-container-service"
+  power       = "nano"
+  scale       = 1
+  is_disabled = false
+}
+
+resource "aws_lightsail_container_service_deployment_version" "whitefox-server" {
+  container {
+    container_name = "whitefox-server"
+    image          = "ghcr.io/agile-lab-dev/io.whitefox.server:latest"
+
+    command = []
+
+    environment = {
+      "WHITEFOX_SERVER_AUTHENTICATION_BEARERTOKEN" = var.whitefox_token
```

Comment:
the current auth behaviour would be:
default is false, token is anyway set to ${WHITEFOX_SERVER_AUTHENTICATION_BEARERTOKEN}
you have to change the application.properties to enable auth
maybe adding ${WHITEFOX_SERVER_AUTHENTICATION_ENABLED} would be good?
so I can default to false, but if its present -> make it true

or I can use only the token as a flag

Reply:
i will add two variables set instead of one :) keep your feature as it is